sys/usb/usbus/hid/hid_io.c

```diff
@@ -57,13 +57,13 @@ void usb_hid_io_write(const void *buffer, size_t len)
     while (len) {
         mutex_lock(&hid.in_lock);
         if (len > max_size) {
-            memmove(buffer_ep + offset, (uint8_t *)buffer + offset, max_size);
+            memmove(buffer_ep, (uint8_t *)buffer + offset, max_size);
             offset += max_size;
             hid.occupied = max_size;
             len -= max_size;
         }
         else {
-            memmove(buffer_ep + offset, (uint8_t *)buffer + offset, len);
+            memmove(buffer_ep, (uint8_t *)buffer + offset, len);
```

Comment:
```suggestion
            memmove(buffer_ep, (uint8_t *)buffer, len);
```
offset seems not needed there too.

Reply:
Why not ? Imagine a buffer of size `CONFIG_USBUS_HID_INTERRUPT_EP_SIZE + 10` then first loop would write `CONFIG_USBUS_HID_INTERRUPT_EP_SIZE` bytes and the second loop would need to write `10` bytes starting at offset `CONFIG_USBUS_HID_INTERRUPT_EP_SIZE`.